charts/victoria-metrics-operator/values.yaml

```diff
@@ -0,0 +1,31 @@
+# Default values for victoria-metrics.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  repository: victoriametrics/operator
+  tag: v0.2.1
+  pullPolicy: IfNotPresent
+imagePullSecrets: []
+nameOverride: "vvvm"
+fullnameOverride: ""
+logLevel: "info"
+rbac:
+  # Specifies whether the RBAC resources should be created
+  create: true
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "her"
+resources: 
+  limits:
+    cpu: 120m
+    memory: 320Mi
+  requests:
+    cpu: 80m
+    memory: 120Mi
+
+nodeSelector: {}
+tolerations: []
+affinity: {}
```

Comment:
may you also add env
```yaml
env:
  []
  # - name: VM_remoteWrite_basicAuth_password
  #   valueFrom:
  #     secretKeyRef:
  #       name: auth_secret
  #       key: password
```

Reply:
i was thinking about it. but do we need to provide any env variables?

Reply:
added